ocaml/xapi/cli_progress_bar.ml

```diff
@@ -22,7 +22,7 @@ module Make(T: Floatable) = struct
     max_value: T.t;
     mutable current_value: T.t;
     width: int;
-    mutable line: string;
+    mutable line: bytes;
```

Comment:
I think we can remove `mutable` now?

Reply:
no, we work on copies and then replace the line member

Reply:
I thought we worked on `t.line` now because we've deleted the `<-` assignment.

Reply:
I think you are right!